packages/jwt/src/jwt-required.hook.ts

```diff
@@ -0,0 +1,66 @@
+// 3p
+import {
+  AbstractUser, Class, Config, Hook, HookDecorator,
+  HttpResponseBadRequest, HttpResponseUnauthorized
+} from '@foal/core';
+import { verify, VerifyOptions } from 'jsonwebtoken';
+import { getRepository } from 'typeorm';
+
+export function JWTRequired(entityClass?: Class<AbstractUser>, options: VerifyOptions = {}): HookDecorator {
+  return Hook(async (ctx, services) => {
+    const secret = Config.get('jwt', 'secret') as string|undefined;
```

Comment:
jwts can also be signed with rsa public/private keys, so having a way to verify against those keys would also be good.

Reply:
Done!